src/components/CMSImage/CMSImage.tsx

```diff
@@ -126,7 +126,7 @@ const CMSImage: FC<CMSImageProps> = (props) => {
           0,
           0,
           originalDimensions.width,
-          originalDimensions.height,
```

Comment:
Looks like old prettier version here... could you merge in main and locally run `npm run format` and commit the changes?